src/components/address-output/_macro.spec.js

```diff
@@ -5,94 +5,141 @@ import * as cheerio from 'cheerio';
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
 
-const EXAMPLE_ADDRESS_OUTPUT_FULL = {
-    unit: 'Unit 5',
-    organisation: 'Trescos',
-    line1: 'Abingdon Road',
-    line2: 'Goathill',
-    town: 'Barry',
-    postcode: 'AB12 6UH',
-};
-
-const EXAMPLE_ADDRESS_OUTPUT_NONE = {};
-
-describe('macro: address-output', () => {
-    it('passes jest-axe checks', async () => {
-        const $ = cheerio.load(renderComponent('address-output', EXAMPLE_ADDRESS_OUTPUT_FULL));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
-    });
+import { EXAMPLE_ADDRESS_OUTPUT_FULL } from './_test_examples';
 
-    it('has additionally provided container style classes', () => {
-        const $ = cheerio.load(
-            renderComponent('address-output', {
-                ...EXAMPLE_ADDRESS_OUTPUT_FULL,
-                classes: 'extra-class another-extra-class',
-            }),
-        );
+describe('FOR: address-output', () => {
```

Comment:
```suggestion
describe('FOR: Macro: Address-output', () => {
```
Same explanation as other PR